### ISSUE TITLE
Example or demo for how to use in project

### ISSUE THREAD

**Issue by kara-todd** [Technohacker/godot_yarn_spinner#2]
This library looks great, and I'd love to leverage it for use in project I'm working on rather than writing yet another parser... :)

However, I was getting a little lost on how to initially connect and import `.yarn` files. My initial import went OK, but then something blew up and now it won't import at all anymore. I also wasn't able to get the events to feed into my dialog box.

Any guidance would be awesome!

**Comment by Technohacker** [comment]
Thanks! I agree I should add an example project too :sweat_smile: 
However for the time being, could I get some more details like any error messages you faced in your project during the import? I did notice a specific bug with Godot where changing the yarn file may not regenerate the translated GDScript file, often printing an error message to the console

**Comment by kara-todd** [comment]
@Technohacker Thanks for responding! :)

Of course, I go to copy the error and now it's working perfectly. :P

I think that it did have something to do with changing a file and re-importing. I wish I had copied it at the time. It might be that I had a malformed file that was borking the parser. If it happens again, I'll post a follow up. I just tried making a modification to the file, and now I see the dialog is updating in the Godot editor with a new `emit_signal` statement. I didn't realize the file is _literally_ transpiled into GDScript. So now trouble shooting errors should be more straitforward.

Once I get a little bit better understanding... I can open a PR for with a demo if that's cool.

**Comment by kara-todd** [comment]
Actually, I think I managed to find one of the errors.

At least one problem I have now is that I wasn't escaping quotes. Here's a simplified example:

<img width="1452" alt="Screen Shot 2020-11-16 at 4 55 29 PM" src="https://user-images.githubusercontent.com/842628/99312806-95b0dc00-282c-11eb-9bec-94369f7d4b47.png">

```
title: Start
tags: 
colorID: 0
position: 738,494
---
person1: Why hello there.
person2: I'd love to have some "cake" with you.
person1: That's terrifying.
===
```

I thought changing [this line](https://github.com/Technohacker/godot_yarn_spinner/blob/master/importer/yarn_to_gd.gd#L67) to the following would fix it... But the import didn't seem to change when I modified it.

```GDScript
static func dialogue(actor, message):
	return [
		"emit_signal(\"dialogue\", self, \"{actor}\", \"{message}\".format(variables))".format({
			"actor": actor.c_escape(),
			"message": message.c_escape()
		}),
		"yield()"
	]
```

The full error is:
```
 built-in:28 - Parse Error: Expected ',' or ')'
 modules/gdscript/gdscript.cpp:599 - Method failed. Returning: ERR_PARSE_ERROR
 Another resource is loaded from path 'res://dialog/simple_test.yarn' (possible cyclic resource inclusion).
```

Forgot to mention, I'm using `Godot Engine v3.2.3.stable.official`

**Comment by Technohacker** [comment]
Oh yeah I don't think I handled quotation marks in the parser, my bad :P

I'll see if I can get that fixed. Also, if you could make a demo project PR, that would be great! :D

**Comment by Technohacker** [comment]
a27f6c3 fixes the bug with quote marks :)

**Comment by kara-todd** [comment]
@Technohacker Thanks so much! I'm getting started on reading the file into a dialog box now that I mostly have it parsing. Once it's working-ish I'll open a PR with a stripped down version for a demo. Might be a bit though ^^;;

**Comment by Technohacker** [comment]
Closed by #4, thanks! :D
